public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -80,24 +80,14 @@ export const SingleTopBar = memo(function SingleTopBar({
             <ToolbarButton iconOnly icon="question-circle" aria-label="Help" />
           </Dropdown>
         )}
-        {config.newsFeedEnabled && <NewsContainer />}
+        {!contextSrv.user.isSignedIn && <SignInLink />}
```

Comment:
was this intentional? i don't think we should change the position of the `Sign in` link

Reply:
no, it should be after kiosk